libs/custom-paper-handler/src/driver/driver.ts

```diff
@@ -23,6 +23,7 @@ import {
   writeImageData,
 } from '@votingworks/image-utils';
 import { Rect } from '@votingworks/types';
+import { Mutex } from '@votingworks/utils';
```

Comment:
This commit is easiest reviewed ignoring whitespace: https://github.com/votingworks/vxsuite/pull/5599/commits/5976abda49a6df6fb492e94a4e1d245a5ac05240?w=1. See `Commit 1: Mutex issues` in the PR description for more details

Reply:
Thanks for upgrading this.